src/Microsoft.DotNet.VersionTools/Automation/VersionsRepoUpdater.cs

```diff
@@ -2,219 +2,34 @@
-                                        }
-                                    }
-                                }
-                            }
-
-                            objects.Add(new GitObject
-                            {
-                                Path = latestPackagesPath,
-                                Type = GitObject.TypeBlob,
-                                Mode = GitObject.ModeFile,
-                                Content = CreatePackageListFile(allPackages)
-                            });
-                        }
-
-                        if (updateLatestVersion)
-                        {
-                            objects.Add(new GitObject
-                            {
-                                Path = $"{versionsRepoPath}/Latest.txt",
-                                Type = GitObject.TypeBlob,
-                                Mode = GitObject.ModeFile,
-                                Content = prereleaseVersion
-                            });
-                        }
-
-                        string message = $"Updating {versionsRepoPath}";
-                        if (string.IsNullOrEmpty(prereleaseVersion))
-                        {
-                            message += ". No prerelease versions published.";
-                        }
-                        else
-                        {
-                            message += $" for {prereleaseVersion}";
-                        }
-
-                        GitTree tree = await client.PostTreeAsync(_project, masterSha, objects.ToArray());
-                        GitCommit commit = await client.PostCommitAsync(_project, message, tree.Sha, new[] { masterSha });
-
-                        // Only fast-forward. Don't overwrite other changes: throw exception instead.
-                        await client.PatchReferenceAsync(_project, masterRef, commit.Sha, force: false);
-
-                        Trace.TraceInformation($"Committed build-info update on attempt {i + 1}.");
-                        break;
-                    }
-                    catch (HttpRequestException ex)
-                    {
-                        int nextTry = i + 1;
-                        if (nextTry < MaxTries)
-                        {
-                            Trace.TraceInformation($"Encountered exception committing build-info update: {ex.Message}");
-                            Trace.TraceInformation($"Trying again in {RetryMillisecondsDelay}ms. {MaxTries - nextTry} tries left.");
-                            await Task.Delay(RetryMillisecondsDelay);
-                        }
-                        else
-                        {
-                            Trace.TraceInformation("Encountered exception committing build-info update.");
-                            throw;
-                        }
-                    }
-                }
-            }
+                .Where(t => !t.SymbolPackage);
```

Comment:
Are there a lot of these?  If so you may consider adding a static method that would take in a package path and return whether or not it pointed to a symbols package.  This could then be called in a where condition before the select clause.